swap_meet/decor.py

```diff
@@ -1,2 +1,16 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    
+    def __init__(self, id=None, width=0, length=0, condition=0):
+        super().__init__(id, condition)
```

Comment:
You could opt to use keyword arguments for `id` and `condition` to ensure that the appropriate values are being passed in as their respective arguments
```suggestion
    def __init__(self, id=None, width=0, length=0, condition=0):
        super().__init__(id=id, condition=condition)
        self.width = width
        self.length = length
```